### PR TITLE
{bio}[foss/2021a] BgeeCall v1.16.0

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,77 @@
+easyblock = 'Bundle'
+
+name = 'BgeeCall'
+version = '1.16.0'
+versionsuffix = '-R-%(rver)s'
+
+# This library does not need last Bioconductor dependencies
+# but better to have its last version from the last Bioconductor version
+local_bioc_version = 3.17
```
[the remark this answers is not in the document]
Comment (smoretti) [reply easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb:9]: It means this library has not requirements linked with the last or a specific Bioconductor release.
So better to use the last Bioconductor version to have up-to-date Bioconductor dependencies.

[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb:9]: I guess you mean that when searching for the Bioconductor extensions, it's better to use the most up to date release, even though we don't need the very latest versions of the extensions?

[the remark this answers is not in the document]
Comment (smoretti) [reply easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb:9]: Exactly

[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb:9]: Okay, I suggest rewording the comment to:
```
# While bGeeCall does not require the latest Bioconductor dependencies
# we search the latest Bioconductor release archive for them
```

[the remark this answers is not in the document]
Comment (smoretti) [reply easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb:9]: Changed

[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/b/BgeeCall/BgeeCall-1.16.0-foss-2021a-R-4.1.0.eb:9]: Thanks.

**Review by verdurin — APPROVED** [review]
Looks fine.
